overlord.pl

```diff
@@ -342,13 +350,35 @@ sub finishCaching
     &waitFor("swapouts gone", sub { ! &squidHasSwapouts() });
 }
 
+sub waitActiveRequests
+{
+    my ($options) = @_;
+
+    my $requestPath = $options->{$onameRequestPath};
+    die("missing $onameRequestPath\n") unless defined $requestPath;
+    my $activeRequestsCount = $options->{$onameActiveRequestsCount};
+    die("missing $onameActiveRequestsCount\n") unless defined $activeRequestsCount;
+
+    my ($path, $count) = @_;
+    &waitFor("until all $activeRequestsCount requests become active", sub { &squidActiveRequests($requestPath, $activeRequestsCount) });
```

Comment:
With all the other changes, this will become something like

```suggestion
    &waitFor("until all $activeRequestsCount requests become active", sub { &countMatchingActiveRequests($requestPath) == $activeRequestsCount });
```

Reply:
Ok.